.github/workflows/push-tests.yml

```diff
@@ -7,6 +7,14 @@ on:
   push:
     branches:
       - "main"
+      - staging
+  repository_dispatch:
+    types: [run-unit-tests]
```

Comment:
I'm not sure we _need_ to run unit tests on staging, since they'll have been run on main.

Reply:
@atolopko-czi - I can remove it. And perhaps it is better to remove it to reduce the complexity.

My thinking, however, is that `staging` branch is the branch from which we will deploy to `prod` and it would be useful to see that we have all our tests passing there. We can certainly revisit this later.